airflow/auth/managers/base_auth_manager.py

```diff
@@ -61,7 +61,7 @@
 
 # This cannot be in the TYPE_CHECKING block since some providers import it globally.
 # TODO: Move this inside once all providers drop Airflow 2.x support.
-ResourceMethod = Literal["GET", "POST", "PUT", "DELETE", "MENU"]
+ResourceMethod = Literal["GET", "POST", "PUT", "PATCH", "DELETE", "MENU"]
```

Comment:
cc: @vincbeck If I'm not mistaken this is not enough to support "PATCH". 
 I think we can edit `_MAP_METHOD_NAME_TO_FAB_ACTION_NAME`, maybe just add an entry "PATCH": `ACTION_CAN_EDIT` is enough. (PATH AND PUT basically will require the same stuff and will be supported)
 
 I assume that aws `VerifiedPermissionsClient` and FAB both support `PATCH` ? (Even if up to now we did not use it)